tests/tests_unit/test_configtools.py

```diff
@@ -289,6 +295,18 @@ def test_read_relative_path():
     assert config.path_field.name == "file.txt"
 
 
+def test_castable_int():
```

Comment:
Looks like this test isn't doing anything, maybe you forgot to add something?

Reply:
Good catch! It was a first draft of a test that I forgot to remove.

Reply:
Deleted it